src/View/Helper/CrudViewHelper.php

```diff
@@ -151,9 +151,12 @@ public function introspect($field, $value, array $options = [])
      */
     public function formatBoolean($field, $value, array $options)
     {
+        if (!array_key_exists('inverted', $options)) {
+            $options['inverted'] = false;
+        }
         return (bool)$value ?
-            $this->Html->label(__d('crud', 'Yes'), 'success') :
-            $this->Html->label(__d('crud', 'No'), 'danger');
+            $this->Html->label(__d('crud', 'Yes'), $options['inverted'] ? 'danger' : 'success') :
```

Comment:
If you do `empty($options['inverted']) ?  'success' :  'danger'`, you won't even need the setting of default value above.